library/std/src/fs.rs

```diff
@@ -31,6 +31,11 @@ use crate::time::SystemTime;
 /// on closing are ignored by the implementation of `Drop`.  Use the method
 /// [`sync_all`] if these errors must be manually handled.
 ///
+/// `File` objects do not buffer reads and writes by default. For efficiency,
```

Comment:
```suggestion
/// `File` does not buffer reads and writes. For efficiency,
```

The "by default" doesn't add anything, it can't be changed. And wrapping it in a buffer makes it a different type.